packages/react/src/__tests__/ReactElementJSXNewWarnings-test.internal.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+let React;
+let ReactDOM;
+
+let ReactFeatureFlags = require('shared/ReactFeatureFlags');
+
+// NOTE: We're explicitly not using JSX here. This is intended to test
+// a new React.jsx api which does not have a JSX transformer yet.
+// A lot of these tests are pulled from ReactElement-test because
+// this api is meant to be backwards compatible.
+describe('ReactElementJSXNewWarnings', () => {
+  let originalSymbol;
+
+  beforeEach(() => {
+    jest.resetModules();
+
+    // Delete the native Symbol if we have one to ensure we test the
+    // unpolyfilled environment.
+    originalSymbol = global.Symbol;
+    global.Symbol = undefined;
+
+    ReactFeatureFlags = require('shared/ReactFeatureFlags');
+    ReactFeatureFlags.warnAboutSpreadingKeyToJSX = true;
```

Comment:
This is already true in the `test-www-variant` run, so instead of moving this to a new file, you can wrap the test in a condition:

```js
if (require('shared/ReactFeatureFlags').warnAboutSpreadingKeyToJSX) {
  // test goes here
}
```

Reply:
See here:
https://github.com/facebook/react/blob/885ed469096d4da7ca038564b323e9b706f5a6d0/packages/shared/forks/ReactFeatureFlags.www-dynamic.js#L10-L18

Reply:
ohh smart